nucypher/characters/control/serializers.py

```diff
@@ -123,7 +123,8 @@ def parse_grant_input(request: dict):
                             label=request['label'].encode(),
                             m=request['m'],
                             n=request['n'],
-                            value=request.get('value'),  # optional / unused in federation mode
+                            value=request.get('value'),  # optional / unused in federated mode
+                            rate=request.get('rate'),  # optional / unused in federated mode
                             expiration=maya.MayaDT.from_iso8601(iso8601_string=request['expiration']))
```

Comment:
Unrelated to this PR, but we need a better UX for specifying this in the CLI.

Reply:
Opened #1606